tests/behavior/transforms/remap.toml

```diff
@@ -1664,10 +1664,17 @@
 [transforms.remap_comments]
   inputs = []
   type = "remap"
-  source = """
+  source = '''
     .a = 1 # .a = 2
     # .a = 3
-  """
+
+    .b = .a == 2 || \
+
+    # We should ignore this too.
+    # .a == 3 && \
+
+    .a == 1
```

Comment:
Does it also work without a comment in-between?

Should we add a test?

```rust
.b = .a == 2 || \

.a == 1
```

Reply:
> Does it also work without a comment in-between?

Have added a test.